base_report_auto_create_qweb/__init__.py

```diff
@@ -1,7 +1,7 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-# For copyright and license notices, see __openerp__.py file in root directory
-##############################################################################
+# -*- coding: utf-8 -*-
+# Authors: See README.RST
+# Copyright 2016 See README.rst for Authors
```

Comment:
That's not standard:

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/__openerp__.py

And by the way you can remove it from `__init__.py` copyrights on imports is a bit overkill